style.css

```diff
@@ -0,0 +1,264 @@
+*, *::after, *::before {
+    box-sizing: border-box;
+} 
+
+body {
+    font-size: 10px;
+    font-family: 'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif;
+    display: flex;
+    flex-direction: column;
+    background-color: #000000;
+}
+
+header {
+
+    background-color: #000000;
+    background-image: url(images/northernLights.jpg);
+    background-size: 90rem 30rem;
+    background-repeat: no-repeat;
+    background-position: center;
+    min-height: 10rem;
+    align-items: baseline;
+    /* padding: 2rem 1rem; */
+}
+
+a {
+    text-decoration: none;
+}
+
+h1 {
+    font-size: 4rem;
+    text-align: center;
+    font-weight: 600;
+}
+
+.navLinks {
+    display: flex;
+    justify-content: center;
+    text-align: center;
+    font-size: 2rem;
+}
+
+.navLinks a {
+    line-height: 2rem;
+    color: #000000;
+    text-decoration: none;
+    padding: 2rem;
+}
+
+.jumbotron {
+    font-size: 1.5rem;
+    /* background-image: url(images/redcircuit.jpg);
+    background-size: 90rem 70rem;
+    background-repeat: no-repeat;
+    background-position: center; */
+    /* I have removed picture temporarily as it clashes */
+    /* background-color: #0e5b1a; */
+    background-color: #000000;
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    padding: 4rem;
+    min-height: 15rem;
+}
+
+.decorativeBox img {
+    min-width: 15rem;
+    min-height: 15rem;
+    max-height: 30rem;
+    max-width: 20rem;
+    border-radius: 200px;
+}
+
+.jumbotronTextbox {
+    min-width: 15rem;
+    min-height: 15rem;
+    max-height: 40rem;
+    max-width: 30rem;
+    margin-left: 4rem;
+}
+
+h2 {
+    color: #ffffff;
+    background-color: #000000;
+}
+
+.jumbotron p {
+    display: inline;
+    font-size: 1.4rem;
+    color: #000000;
+    background-color: #1B8C20;
+    line-height: 2.8rem;
+}
+
+.jumbotronTextbox button {
+    margin: 4rem;
+    padding: 1rem;
+    border-radius: 25px;
+    border: 0.5rem solid #1B8C20;
+    color: #ffffff;
+    background-color: #000000;
+}
+
+.aboutMe {
+    /* background-color: #206D27; */
+    /* background-color: #0DBD29; */
+    background-color: #1B8C20;
+    font-size: 2rem;
+    margin-left: 6rem;
+    margin-right: 6rem;
+
+}
+
+.aboutMe h2 {
+    padding-left: 1.6rem;
+}
+
+.aboutMe p {
+    padding: 1.6rem;
+    line-height: 1.7;
+}
+
+.portfolioBlurb {
+    background-color: #1B8C20;
+    font-size: 2rem;
+    margin-left: 6rem;
+    margin-right: 6rem;
+}
+
+.portfolioBlurb h2 {
```

Comment:
This is the same as aboutMe h2 a few lines above. Just combine to reduce duplicate code:
```
.aboutMe h2, .portfolioBlurb h2 {
//stuff
}
```